setup/setup.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+
+ORKA_VM_TOOLS_VERSION="${ORKA_VM_TOOLS_VERSION:-3.5.0}"
+CURRENT_USER="${USER}"
+
+GREEN="\033[0;32m"
+YELLOW="\033[1;33m"
+RED="\033[0;31m"
+NC="\033[0m"
+
+set -euo pipefail
+
+log() {
+    echo -e "${GREEN}[INFO]${NC} $1"
+}
+
+warn() {
+    echo -e "${YELLOW}[WARN]${NC} $1"
+}
+
+error() {
+    echo -e "${RED}[ERROR]${NC} $1" >&2
+    exit 1
+}
+
+if [[ $EUID -eq 0 ]]; then
+    error "This script should not be run as root. Please run as a regular user with sudo privileges."
+fi
+
+if ! command -v sudo &> /dev/null; then
+    error "sudo is required but not available"
+fi
+
+log "Starting MacOS Orka VM setup..."
+
+enable_screen_sharing() {
+    log "Enabling Screen Sharing..."
+    
+    sudo launchctl load -w /System/Library/LaunchDaemons/com.apple.screensharing.plist
+    
+    log "Screen Sharing enabled"
+}
+
+enable_remote_login() {
+    log "Enabling Remote Login (SSH)..."
+   
+    sudo launchctl enable system/com.openssh.sshd
+    sudo launchctl load -w /System/Library/LaunchDaemons/ssh.plist 2>/dev/null || true
+    
+    log "Remote Login (SSH) enabled"
+}
+
+# Configure macOS updates
+configure_macos_updates() {
```

Comment:
This still does not work for me btw. Updates are still enabled if I check in System Preferences after running the script.

If we cannot figure this out - I would suggest just to remove it. As it is not strictly required

Reply:
Removed this step, will review the readme and update the docs to clarify system updates etc. for Tahoe